meson.build

```diff
@@ -0,0 +1,845 @@
+project('eigen3', 'cpp', version: '3.3.4', lincese: 'MPL2')
```

Comment:
Correct keyword is `license`. See [project()](http://mesonbuild.com/Reference-manual.html#project).